toolset/setup/linux/languages/rvm.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-RETCODE=$(fw_exists rvm.installed)
+RETCODE=$(fw_exists ${IROOT}/rvm.installed)
```

Comment:
Thanks. I like to figure out what causes Ruby frameworks failing, but I am assigned to another project, so I just report this fact here.